react/useCustomEvents.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useRef } from 'react'
+
+const useCustomEvent = (
```

Comment:
What do you think of:

```suggestion
const usePixelEvent = (
```

Reply:
What about?

```tsx
usePixelEventCallback('addToCart', (event) => {
  doSomething()
})
```